.github/workflows/spellchecker.yml

```
@@ -0,0 +1,12 @@
+name: Spell checking
```

Other actions don't start with a capital letter

```suggestion
name: spell checking
```